http_service/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.2'
```

Comment:
Can we remove the http service stuff from the root docker-compose and only leave it here?

Reply:
If we do that, we won't build the Docker http image during release or we would have to also run `taskboot build-compose` against this docker-compose.

We still need to build the bg worker image